app/routes/events/view/index.js

```diff
@@ -1,11 +1,20 @@
 import Route from '@ember/routing/route';
+import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
 
-export default Route.extend({
-  async model() {
+export default class extends Route.extend(EmberTableRouteMixin) {
+
+  async model(params) {
     let eventDetails = this.modelFor('events.view');
+    const searchField = 'name';
+    let filterOptions = [];
+    filterOptions = this.applySearchFilters(filterOptions, params, searchField);
     let sponsorQueryObject = {
```

Comment:
why name it this, when every other table in the app has the name `queryString`, you just have to copy, it makes it inconsistent, though not incorrect

Reply:
To maintain the consistency as before of `Ember Table`, It was named `SponsorQueryObject` in `Model Table` too

Reply:
but in each single ember table, it's queryString, so please rename it to that.